src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/auditing/LogOutAuditHandler.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.auditing
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.security.core.Authentication
+import org.springframework.security.web.authentication.logout.LogoutSuccessHandler
+import uk.gov.justice.digital.hmpps.pecs.jpc.service.AuditService
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+
+class LogOutAuditHandler : LogoutSuccessHandler {
```

Comment:
Change to pass the service in via the constructor

`class LogOutAuditHandler(private val auditService: AuditService) : LogoutSuccessHandler {
`